pkg/oktaApiAuth/oktaApiAuth.go

```diff
@@ -74,17 +70,24 @@ func (auth *OktaApiAuth) verifyFactors(stateToken string, factors []AuthFactor,
 			return nil
 		}
 
-		if count == nbFactors-1 {
-			log.Error().Msgf("%s %s MFA authentication failed: %s",
+		var mfaErr error
+		if authRes.Status != "" {
```

Comment:
```suggestion
		if authRes.Result != "" {
```

Not sure, but it makes more sense to me 😅